crypto/dh/dh_ameth.c

```diff
@@ -185,6 +187,7 @@ static int dh_priv_decode(EVP_PKEY *pkey, const PKCS8_PRIV_KEY_INFO *p8)
     if (!DH_generate_key(dh))
         goto dherr;
 
+    dh->dirty_cnt++;
```

Comment:
Hasn't the `dirty_cnt` already been incremented by the `DH_generate_key()` call?

Reply:
Ah, yes it does.  Thank you.  ...  not that it really matters, but yeah